sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionLiveTest.cs

```diff
@@ -39,7 +39,7 @@ private LogsIngestionClient CreateClient(HttpPipelinePolicy policy = null)
         public void NullInput()
         {
             LogsIngestionClient client = CreateClient();
-            Assert.ThrowsAsync<ArgumentNullException>(async () => await client.UploadAsync(TestEnvironment.DCRImmutableId, TestEnvironment.StreamName, null).ConfigureAwait(false));
+            Assert.ThrowsAsync<NullReferenceException>(async () => await client.UploadAsync(TestEnvironment.DCRImmutableId, TestEnvironment.StreamName, null).ConfigureAwait(false));
```

Comment:
We shouldn't need to change this test - this suggests that we are missing some null validation now that we are not calling the generated method.